### PR TITLE
Add optional `as_of_date` for live debugging assessment-status

### PR DESCRIPTION
**Pull request by pbugni**
Found this missing when debugging a prod assessment issue.

### CODE REVIEW

**Comment by ivan-c** [comment]
@pbugni Should this be part of this week's release?

File: portal/views/assessment_engine.py [hunk 1]
```diff
@@ -1788,12 +1794,14 @@ def patient_assessment_status(patient_id):
     patient = get_user_or_abort(patient_id)
     current_user().check_role(permission='view', other_id=patient_id)
 
-    now = datetime.utcnow()
+    date = request.args.get('as_of_date')
+    # allow date and time info to be available
+    date = FHIR_datetime.parse(date) if date else datetime.utcnow()
```
Comment (ivan-c) [inline portal/views/assessment_engine.py]: Why not set (and pass) to `None`, so `QB_Status()` always sets the default?

Comment (pbugni) [reply portal/views/assessment_engine.py]: oh, clarity.  no reason.

I'll change, and recreate the PR to hit the RC.

**Comment by pbugni** [comment]
Closing in favor of new PR to merge into RC

**Pull request closed without merging**